### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier MOD"
### and CLHS topic selection http://clhs.lisp.se/Body/a_mod.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:mod.txt	2017-04-27 21:49:41.000000000 +0100
+++ saved-pages-clus/cl:types:mod.txt	2017-04-27 21:49:41.000000000 +0100
@@ -1,4 +1,36 @@
-MOD
-Please select which reference to MOD you intended
+Type Specifier MOD

-FunctionType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Abbreviating.
+Compound Type Specifier Syntax
+ mod n
+Compound Type Specifier Arguments
+ n - a positive integer.
+Compound Type Specifier Description
+This denotes the set of non-negative integers less than n. This is equivalent to (integer 0 (n)) or to (integer 0 m), where m = n - 1.
+The argument is required, and cannot be *.
+The symbol mod is not valid as a type specifier.
+\
\ No newline at end of file
